### PR TITLE
Add function for formatting numbers in written form

### PR DESCRIPTION
**Pull request by phalkunz**
Issue: https://github.com/silverstripe/silverstripe-framework/issues/6685

### CODE REVIEW

File: client/lang/en.js [hunk 1]
```diff
@@ -29,6 +29,16 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "Admin.LINKLABEL_EXTERNALURL": "Link to external URL",
     "Admin.LINKLABEL_EMAIL": "Link to email address",
     "Admin.LINK_EXTERNAL": "Insert external link",
-    "Admin.LINK_EMAIL": "Insert email link"
+    "Admin.LINK_EMAIL": "Insert email link",
+    "Admin.WRITTEN_NUMBER_ZERO": "zero",
```
Comment (phalkunz) [inline client/lang/en.js]: Not quite sure what you mean. Is there an example I can see.

File: client/src/styles/_layout.scss [hunk 2]
```diff
@@ -113,3 +113,15 @@ body {
...
 }
+
+
+/**
+ * Workaround for tooltip placement. React bootstrap add class `left`, `right`,
+ * `top` and `bottom` instead of `tooltip-left`, `tooltip-right`, `tooltip-top`
+ * and `tooltip-bottom`.
+ */
+@each $dir in [left, right, top, bottom] {
+  .tooltip.#{$dir} {
```
Comment (phalkunz) [inline client/src/styles/_layout.scss]: But there isn't a tooltip component? Will it be just scss file in the folder?

File: client/lang/en.js (hunk 1 above)
Comment (flamerohr) [inline client/lang/en.js]: So you'll find a list of translations (with code surrounding this list) named `client/lang/en.js`
You'll also find another file named `client/lang/src/en.json` which contains the same list of translations (but without the code surrounding the list)

File: client/src/styles/_layout.scss (hunk 2 above)
Comment (flamerohr) [inline client/src/styles/_layout.scss]: Yeah, that's fine :)
We had something similar with `Label/Label.scss` (actually, that could be cleaned up to be `LabelField/LabelField.scss` later on)
And I think it's better practice to have component specific styles handled like a component specific style, even if we don't have a javascript component for it.

Comment (phalkunz) [inline client/src/styles/_layout.scss]: 👍

File: client/lang/en.js (hunk 1 above)
Comment (phalkunz) [inline client/lang/en.js]: I see. Thanks :)

**Pull request merged**